oxidauth-kernel/src/rsa/mod.rs

```diff
@@ -129,28 +129,28 @@ impl TryFrom<&[u8]> for PrivateKey {
     }
 }
 
-#[cfg(test)]
-#[cfg(feature = "rsa-test")]
-mod tests {
-    use super::*;
+// #[cfg(test)]
```

Comment:
??

Reply:
These tests were causing warnings in the code editor because they were not being used. Feature rsa not specified in library. Commenting them out got rid of the warning

Reply:
Warning is fine. It's because they aren't a part of the main set of tests but we still want to be able to run these. 

The reason the aren't part of the tests is because they are kinda slow to run.